openshift/templates/images/postgres/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:15.3-bullseye
+FROM postgres:15.4-bullseye
```

Comment:
Interesting that this was using bullseye (Debian) instead of Alpine like the node images. A postgres:15.4-alpine3.18 image does exist: https://hub.docker.com/layers/library/postgres/15.4-alpine3.18/images/sha256-f36c528a2dc8747ea40b4cb8578da69fa75c5063fd6a71dcea3e3b2a6404ff7b?context=explore

Reply:
I could change it - @ShawnTurple does it matter to you either way?

Reply:
bullseye is what is being used in OpenShift, and this is right.